### ISSUE TITLE
[Bug]: [Nightly] Load collection failed because collection with different replica number 1 existed

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#21023]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: f8cff79
- Deployment mode(standalone or cluster):cluster
- SDK version(e.g. pymilvus v2.0.0rc2):2.3.0.dev15
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Load collection failed because collection with different replica number 1 existed
```
[2022-12-06T19:24:18.499Z] [2022-12-06 17:59:57 - ERROR - pymilvus.decorators]: RPC error: [load_collection], <MilvusException: (code=5, message=failed to load collection, err=collection with different replica number 1 existed, release this collection first before changing its replica number[LoadParameterMismatched])>, <Time:{'RPC start': '2022-12-06 17:59:57.525512', 'RPC error': '2022-12-06 17:59:57.531576'}> (decorators.py:108)
[2022-12-06T19:24:18.499Z] [2022-12-06 17:59:57 - ERROR - ci_test]: Traceback (most recent call last):
[2022-12-06T19:24:18.499Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 26, in inner_wrapper
[2022-12-06T19:24:18.499Z]     res = func(*args, **_kwargs)
[2022-12-06T19:24:18.499Z]   File "/home/jenkins/agent/workspace/tests/python_client/utils/api_request.py", line 57, in api_request
[2022-12-06T19:24:18.499Z]     return func(*arg, **kwargs)
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/orm/collection.py", line 361, in load
[2022-12-06T19:24:18.499Z]     conn.load_collection(self._name, replica_number=replica_number, timeout=timeout, **kwargs)
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 109, in handler
[2022-12-06T19:24:18.499Z]     raise e
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 105, in handler
[2022-12-06T19:24:18.499Z]     return func(*args, **kwargs)
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 136, in handler
[2022-12-06T19:24:18.499Z]     ret = func(self, *args, **kwargs)
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 85, in handler
[2022-12-06T19:24:18.499Z]     raise e
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/decorators.py", line 50, in handler
[2022-12-06T19:24:18.499Z]     return func(self, *args, **kwargs)
[2022-12-06T19:24:18.499Z]   File "/usr/local/lib/python3.8/site-packages/pymilvus/client/grpc_handler.py", line 656, in load_collection
[2022-12-06T19:24:18.499Z]     raise MilvusException(response.error_code, response.reason)
[2022-12-06T19:24:18.499Z] pymilvus.exceptions.MilvusException: <MilvusException: (code=5, message=failed to load collection, err=collection with different replica number 1 existed, release this collection first before changing its replica number[LoadParameterMismatched])>
[2022-12-06T19:24:18.499Z]  (api_request.py:39)
[2022-12-06T19:24:18.499Z] [2022-12-06 17:59:57 - ERROR - ci_test]: (api_response) : <MilvusException: (code=5, message=failed to load collection, err=collection with different replica number 1 existed, release this collection first before changing its replica number[LoadParameterMismatched])> (api_request.py:40)
```

### Expected Behavior

pass

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/212/pipeline/155
```

### Milvus Log

1. log: [artifacts-milvus-distributed-kafka-nightly-212-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/212/artifact/artifacts-milvus-distributed-kafka-nightly-212-pymilvus-e2e-logs.tar.gz)
2. collection name: collection_2EB4ylmx
3. failed time: [2022-12-06T18:00:03.488Z] [gw2] [  9%] FAILED testcases/test_collection.py::TestLoadCollection::test_load_replica_change

**Comment by NicoYuan1986** [comment]
Still failing.
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/213/pipeline/155
case: test_load_replica_change

**Comment by sunby** [comment]
Same as https://github.com/milvus-io/milvus/issues/21000

**Comment by sunby** [comment]
@NicoYuan1986 Fixed in #21069

**Comment by NicoYuan1986** [comment]
Fixed.
